modules/test/playwright/tests/frontend-data-set-admin-web/utils/clickActionInRow.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect} from '@playwright/test';
+
+export default async function clickActionInRow({
+	actionName,
+	rowName,
+	visualizationModesPage,
```

Comment:
:+1: for moving this to a util. But as this is a util, we should generalize `visualizationModesPage` into `page`. Later on we may invoke it from some other page.